tests/test_scheduler.py

```diff
@@ -714,8 +717,11 @@ def test_full_loop_no_noise(self):
         result_sum = torch.sum(torch.abs(sample))
         result_mean = torch.mean(torch.abs(sample))
 
-        assert abs(result_sum.item() - 14379591680.0) < 1e-2
```

Comment:
same here - we're getting large differences because we don't pass a generator. 
Let's maybe merge this PR: https://github.com/huggingface/diffusers/pull/289 first and then pass a generator here

Reply:
ok